java.lang.NoSuchMethodError: 'java.lang.String javax.annotation.Resource.lookup()'

```
Caused by: java.lang.NoSuchMethodError: 'java.lang.String javax.annotation.Resource.lookup()'
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor$ResourceElement.<init>(CommonAnnotationBeanPostProcessor.java:664)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.lambda$buildResourceMetadata$0(CommonAnnotationBeanPostProcessor.java:395)
	at org.springframework.util.ReflectionUtils.doWithLocalFields(ReflectionUtils.java:677)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.buildResourceMetadata(CommonAnnotationBeanPostProcessor.java:377)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.findResourceMetadata(CommonAnnotationBeanPostProcessor.java:358)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.postProcessMergedBeanDefinition(CommonAnnotationBeanPostProcessor.java:306)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyMergedBeanDefinitionPostProcessors(AbstractAutowireCapableBeanFactory.java:1116)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:594)
```
RocketMQ(Java) Version: 5.0.0
Spring Version: 5.3.22
Spring Boot Version: 2.7.3

--- a/java/client-shade/pom.xml
+++ b/java/client-shade/pom.xml
@@ -41,6 +41,10 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>rocketmq-client-java-noshade</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.tomcat</groupId>
+            <artifactId>annotations-api</artifactId>
+        </dependency>
     </dependencies>
 
     <build>
@@ -56,6 +60,11 @@
                             <goal>shade</goal>
                         </goals>
                         <configuration>
+                            <artifactSet>
+                                <excludes>
+                                    <exclude>org.apache.tomcat:annotations-api</exclude>
+                                </excludes>
+                            </artifactSet>
                             <minimizeJar>false</minimizeJar>
                             <createSourcesJar>true</createSourcesJar>
                             <shadeSourcesContent>true</shadeSourcesContent>

--- a/java/client/pom.xml
+++ b/java/client/pom.xml
@@ -45,6 +45,10 @@
             <artifactId>rocketmq-proto</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.tomcat</groupId>
+            <artifactId>annotations-api</artifactId>
+        </dependency>
+        <dependency>
             <groupId>io.grpc</groupId>
             <artifactId>grpc-netty-shaded</artifactId>
         </dependency>

--- a/java/pom.xml
+++ b/java/pom.xml
@@ -48,6 +48,7 @@
            ~  2. Make sure that it is compatible with Java 8.
          -->
         <rocketmq-proto.version>2.0.0</rocketmq-proto.version>
+        <annotations-api.version>6.0.53</annotations-api.version>
         <protobuf.version>3.19.2</protobuf.version>
         <grpc.version>1.47.0</grpc.version>
         <guava.version>31.1-jre</guava.version>
@@ -118,7 +119,16 @@
                         <groupId>io.grpc</groupId>
                         <artifactId>grpc-netty-shaded</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.tomcat</groupId>
+                        <artifactId>annotations-api</artifactId>
+                    </exclusion>
                 </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.tomcat</groupId>
+                <artifactId>annotations-api</artifactId>
+                <version>${annotations-api.version}</version>
             </dependency>
             <dependency>
                 <groupId>com.google.protobuf</groupId>

